### ISSUE TITLE
Search from predefined searches

### ISSUE THREAD

**Issue by dani-h** [tmux-plugins/tmux-copycat#86]
First of all this is a very handy plugin, so thank you to the author.

I'm currently using the git status predefined search and occasionally the file search. The only deficiency I have with these two predefined searches is that when you need to jump over a long list of git status files it takes a lot of clicks. It would be very handy if you could search the set of changed files according to git using a few letters instead of jumping over all files until you find your match. Is this possible?

**Comment by bruno-** [comment]
Agreed, I would also find this feature useful. However, this is an enhancement and I don't have the time to tackle it right now :/

PR for this would be welcome.

The solution will probably need to filter (remove) lines from the output of [this function](https://github.com/tmux-plugins/tmux-copycat/blob/33b98809a5b46fecc8bb48f0e3ccd801689981e4/scripts/copycat_git_special.sh#L9-L13). That function returns all `git status` files, but for this feature we want it to return only files containing letters we entered. Essentially this: `git_status_files | grep 'my_file_name'`
